upgrade.sh

```diff
@@ -108,6 +108,35 @@ fi
 undeploy -n services cray-conman
 
 # Deploy remaining system management applications
+
```

Comment:
I think we need to refactor this part of the upgrade script for readability.

At a minimum some of this should be annotated, but I think we also need to look at adding error handling of some kind.